nix/shell.sh

```diff
@@ -16,12 +16,14 @@ function clean {
 }
 
 function configure {
+    local pq=${1:-0}
```

Comment:
Do these scripts respect env vars? Could you do `S2N_NO_PQ=1 configure` instead of `configure 1`?

Reply:
Pulling this out, I think the environment vars approach is certainly more readable.